docs/running-node-ansible/genesis-vn.md

```diff
@@ -0,0 +1,75 @@
+# Running Genesis Validator Node using Ansible
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+
+## Configure DCL network parameters (local machine)
+### 1. Set network chain id in [`deployment/ansible/inventory/hosts.yml`]
```

Comment:
Probably better to have `[`deployment/ansible/inventory/hosts.yml`]` on the next line as in other headers